### ISSUE TITLE
How fast?

### ISSUE THREAD

**Issue by pickfire** [rbatis/fast_log#2]

**Comment by zhuxiujia** [comment]
QPS: 525892 Logs/s         with  win10(PC 6core,Mechanical drive)

**Comment by pickfire** [comment]
Compared to log?

**Comment by zhuxiujia** [comment]
> How about when compared to log? Oh, or should this be compared to simplelog? How fast is it when compared to the alternatives?

Not surprisingly, it should be the fastest. Because the architecture is to write memory before writing to disk, and is suitable for WAL disk

**Comment by pickfire** [comment]
Yeah, everyone could argue that it is faster but at least a benchmark is required to do the comparison, I mean even in least. I thought https://github.com/pickfire/ve would be faster too but it isn't.

**Comment by zhuxiujia** [comment]
> Yeah, everyone could argue that it is faster but at least a benchmark is required to do the comparison, I mean even in least. I thought https://github.com/pickfire/ve would be faster too but it isn't.

so  you can add test benchmark  with other log lib  if you want。
